coaster/views.py

```diff
@@ -574,3 +574,41 @@ def decorated_function(*args, **kwargs):
                 return result
         return decorated_function
     return inner
+
+def cors(is_origin_allowed=lambda origin: False,
```

Comment:
Make the parameter mandatory. If it defaults to False, making the method uncallable, there's no reason to have the default.